deployment/scripts/upgrade.sh

```diff
@@ -0,0 +1,470 @@
+#! /bin/bash
+
+set -ex
+
+TIMES=300
+
+fluentd_nodeselector="logging-infra-fluentd=true"
+
+function getDeploymentVersion() {
+  #base this on what isn't installed
+  local version=0
+
+  # Check for the admin cert
+  if [[ -n "$(oc get secrets -o jsonpath='{.items[?(@.data.admin-cert)].metadata.name}')" ]]; then
+    version=1
+  else
+    echo "$version"
+    return
+  fi
+
```

Comment:
Could shorten above to this:

    if [[ -z "$(oc get secrets -o jsonpath='{.items[?(@.data.admin-cert)].metadata.name}')" ]]; then
      echo 0
      return
    fi

and similarly below